tests/python/gaia-ui-tests/gaiatest/tests/functional/music_oga/manifest.ini

```diff
@@ -0,0 +1,26 @@
+[parent:../manifest.ini]
```

Comment:
I'm sure we probably would deprecate it eventually, but I thought perhaps we should maintain it at least until 2.5 is released, since if we just backout this commit, any fixes we (might) do on oga test would be lost.